### ISSUE TITLE
Add h3 to balloon widget

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/balloons-and-static-electricity#171]
Asked in design doc, but adding here to document the question.

Sweater and wall are both labelled with an h3.  For consistency, should the balloon widget also be labelled as such?  Perhaps this would add some verbocity, a screen reader might say

'Heading level 3, Yellow balloon' [down arrow]
'Grab yellow balloon, button' [down arrow]
'Once grabbed, get ready to...' [down arrow]

But if we use headings to deliver structure information, it might seem that the wall and sweater are more important than the balloons.  What do you think @terracoda?

**Comment by terracoda** [comment]
@jessegreenberg, I totally agree. The balloons' names definitely need to be in a heading at the same level as the Sweater and the Wall (h3). The balloon must be part o the heading hierarchy :-). I was just wondering if the button should come before or after the heading. I was originally thinking it should come before, but after is also good. The balloon description, however, should immediately follow the heading, I think.

What do you think of this order?

'Heading level 3, Yellow balloon' [down arrow]
Paragraph containing: 'In center of play area. Has a net neutral charge, no more negative charges than positive ones. Look for grab button to play.' [down arrow]
'Grab yellow balloon, button' [down arrow]
'Once grabbed, get ready to...' [down arrow]

**Comment by jessegreenberg** [comment]
@terracoda, yes I like that a lot! This way the information about the balloon should be very easy to find, perhaps the user will have more context before getting to information about the interaction.  It also matches the sweater and wall.

**Comment by jessegreenberg** [comment]
Assigning to myself to reorder.

**Comment by jessegreenberg** [comment]
So maybe the html (some attributes missing!) would look something like:
```html
<div id='widget-container'>
	<h3>Yellow Balloon</h3>
	<p id='balloon-description'>In center of play area.  Has a net neutral charge, no...</p>
	<div id='button-container'>
		<button>Grab Yellow Balloon</button>
		<p>Once grabbed, get ready to...</p>
	</div>
	<div id='draggable-balloon' role='application' aria-describedby='balloon-description'></div>
</div>
```

So that the description is still associated with the draggable element, and the user will get an up to date description of the balloon once it is grabbed?

**Comment by terracoda** [comment]
@jessegreenberg, this looks good to me. I like that the button will be associated under the balloon heading, really making structurally part of the balloon which it is, in my mind.

**Comment by jessegreenberg** [comment]
Thanks @terracoda, this should be done in the commit above. Closing.
